internal/webhooks/metal3.io/v1alpha1/baremetalhost_validation.go

```diff
@@ -386,3 +390,13 @@ func validatePowerStatus(host *metal3api.BareMetalHost) error {
 	}
 	return nil
 }
+
+func validateLiveISO(bmcAddress string, image *metal3api.Image) error {
+	if image != nil &&
+		image.DiskFormat != nil &&
+		*image.DiskFormat == "live-iso" &&
+		!strings.Contains(strings.Split(bmcAddress, "://")[0], "virtualmedia") {
```

Comment:
Can we use something like `hasPrefix("redfish-virtualmedia://")` , seems more robust?

Reply:
Do we have a list somewhere over prefixes? because I found these in the docs:
- redfish-virtualmedia://
- idrac-virtualmedia://
- ilo5-virtualmedia://
- ilo4-virtualmedia://
and all of there can also have +http or +https.
I can probably use hasPrefix instead yeah

Reply:
I'm good with this iteration.